src/main/java/com/bittercode/service/impl/UserServiceImpl.java

```diff
@@ -37,10 +42,11 @@ public User login(UserRole role, String email, String password, HttpSession sess
             ps.setString(3, userType);
             ResultSet rs = ps.executeQuery();
             if (rs.next()) {
+                //Setter methods will never set a value as null (check implementation)
                 user = new User();
                 user.setFirstName(rs.getString("firstName"));
-                user.setLastName(rs.getString("lastName"));
-                user.setPhone(rs.getLong("phone"));
+                user.setLastName(rs.getString( "lastName"));
```

Comment:
the added spaces here are another example of an unintended change that should be removed before the review stage